File: jdroid-java-core/src/main/java/com/jdroid/java/gson/GsonBuilderFactory.java
package com.jdroid.java.gson;

import com.google.gson.GsonBuilder;
import com.jdroid.java.date.DateConfiguration;

public class GsonBuilderFactory {

	public static GsonBuilder createGsonBuilder() {
		GsonBuilder gsonBuilder = new GsonBuilder();
		gsonBuilder.setDateFormat(DateConfiguration.getDefaultDateTimeFormat());
		gsonBuilder.disableHtmlEscaping();
		return gsonBuilder;
	}
}

File: jdroid-java-core/src/main/java/com/jdroid/java/gson/GsonBuilderFactory.kt
package com.jdroid.java.gson

import com.google.gson.GsonBuilder
import com.jdroid.java.date.DateConfiguration

object GsonBuilderFactory {

    fun createGsonBuilder(): GsonBuilder {
        val gsonBuilder = GsonBuilder()
        gsonBuilder.setDateFormat(DateConfiguration.getDefaultDateTimeFormat())
        gsonBuilder.disableHtmlEscaping()
        gsonBuilder.setExclusionStrategies(HiddenAnnotationExclusionStrategy())
        return gsonBuilder
    }
}
